### PR TITLE
Creating a draft blog post

### PR DESCRIPTION
**Pull request by ms-sambell**
Drafting a blog post for us to publish. 

This is a first draft and needs a bit of work, but it's a start.

### CODE REVIEW

File: blogPosts/psrules-june-2022.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+PsRules for Azure can be installed locally by running the following commands in a PowerShell terminal:
+
+To install PSRule for Azure for the current user use:
+
+    ```powershell
+    Install-Module -Name 'PsRule' -Repository PSGallery -Scope CurrentUser -Force
+    Install-Module -Name 'PSRule.Rules.Azure' -Repository PSGallery -Scope CurrentUser -Force
+    ```
+
+To enable these new modules for the current user use:
```
Comment (BernieWhite) [inline blogPosts/psrules-june-2022.md]:
```suggestion
To update these modules for the current user use:
```

File: blogPosts/psrules-june-2022.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+    ```powershell
+    Update-Module -Name 'PSRule'
+    Update-Module -Name 'PSRule.Rules.Azure'
+    ```
```
Comment (BernieWhite) [inline blogPosts/psrules-june-2022.md]: Markdown formatting is not correct. Either remove the language identifier or remove the indent to fix.

File: blogPosts/psrules-june-2022.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+#### Step 1 - Installation
+
+PsRules for Azure can be installed locally by running the following commands in a PowerShell terminal:
+
+To install PSRule for Azure for the current user use:
+
+    ```powershell
+    Install-Module -Name 'PsRule' -Repository PSGallery -Scope CurrentUser -Force
+    Install-Module -Name 'PSRule.Rules.Azure' -Repository PSGallery -Scope CurrentUser -Force
+    ```
```
Comment (BernieWhite) [inline blogPosts/psrules-june-2022.md]: Markdown formatting is not correct. Either remove the language identifier or remove the indent to fix.

File: blogPosts/psrules-june-2022.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
+# PsRules - Blog Post
```
Comment (BernieWhite) [inline blogPosts/psrules-june-2022.md]:
```suggestion
# PSRule - Blog Post
```

File: blogPosts/psrules-june-2022.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+| Advanced | Validate IAC against your organisations standards | Each organisation has a different set of recommendations and standards. However it can be challenging to enforce these standards. For the most part these standards are either enforced in the IAC template or just documented. What if there was a tool that allows you to create rules to help enforce these standards (for free?!) | ARM & Bicep: [PSRules](https://microsoft.github.io/PSRule/v2/quickstart/standalone-rule/). |
+
+In this blog post we're going to focus on using **PsRules for Azure**. A blog post will be written in the future for **PsRules**, however for more information, please use the link provided below.
```
Comment (BernieWhite) [inline blogPosts/psrules-june-2022.md]:
```suggestion
In this blog post we're going to focus on using **PSRule for Azure**. A blog post will be written in the future for **PSRule**, however for more information, please use the link provided below.
```

File: blogPosts/psrules-june-2022.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+    }
+    ```
+
+This file is available locally on my device in a folder called `examples`. Before we can test the Bicep module we need to test create a file called `ps-rule.yaml` with the following content:
+
+    ```yaml
+    # YAML: Enable expansion for Bicep source files.
+    configuration:
+    AZURE_BICEP_FILE_EXPANSION: true
+    ```
```
Comment (BernieWhite) [inline blogPosts/psrules-june-2022.md]:
```suggestion
    ```yaml
    # YAML: Enable expansion for Bicep source files.
    configuration:
      AZURE_BICEP_FILE_EXPANSION: true
    ```
```

**Comment by ms-sambell** [comment]
@ArmaanMcleod Did you have any chances you'd like me to include?

**Comment by ArmaanMcleod** [comment]
> @ArmaanMcleod Did you have any chances you'd like me to include?

@ms-sambell I think adding a section for making custom rules would be a good idea. Doesn't have to be too advanced.

Would be good to show the reader how they can setup their own rules if needed.

**Comment by ArmaanMcleod** [comment]
Also happy to add this in the next blog post.

**Pull request merged**